src/CanvasItem/CanvasItem.js

```diff
@@ -219,7 +245,9 @@ CanvasItem.propTypes = {
    */
   width: PropTypes.number.isRequired,
   /**
-   * The zoomable direction of the item. It can be any subset of "n, ne, e, se, s, sw, w, nw" and is provides as a comma delimited string.
+   * The zoomable direction of the item. Array of 'n', 's', 'e', 'w', 'ne', 'nw', 'se', 'sw'.
    */
-  zoomable: PropTypes.string
+  zoomable: PropTypes.arrayOf(
```

Comment:
Sorry to be picky but zoomable is not a good prop name here. When I read it I'm expecting it to be a boolean, like rotatable is. Also, it's nothing to do with zooming. It's to do with resizing. I would name it the same as react-resizable which we are replacing here. They use 'resizeHandles' as the prop name.

Reply:
'resizeDirection' would also be a good name.

Reply:
This was actually what I meant by https://github.com/IPG-Automotive-UK/react-ui/pull/461#issuecomment-1275803159. Sorry for not being clearer.